generators/entity/templates/src/main/java/package/service/impl/_EntityServiceImpl.java

```diff
@@ -95,8 +95,14 @@ public void delete(<%= pkType %> id) {
      *  @param query the query of the search
      *  @return the list of entities
      */<% if (databaseType == 'sql') { %>
-    @Transactional(readOnly = true) <% } %>
-    public List<<%= instanceType %>> search(String query) {
-        <%- include('../../common/search_template', {viaService: viaService}); -%>
+    @Transactional(readOnly = true)<% } %>
```

Comment:
I can do that but I'll need to split up search_template. Should we refactor get_all_template too while we're at it?